app/src/main/java/com/xa/xpensauditor/TabFragment.java

```diff
@@ -139,6 +186,11 @@ public void OnItemClick(int position, View v) {
                 startActivity(i);
             }
 
+            /**
+             * On a long clock, view position of the transaction in the Fragment.
+             * @param position
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.NonEmptyAtclauseDescriptionCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>At-clause should have a non-empty description.